### DISCUSSION TITLE
Hello everybody! How to open flet app on Telegram web view? I am using aiogram.

### DISCUSSION THREAD

**Discussion by qahor0v** [flet-dev/flet#2151] [Q&A]

**Comment by qahor0v** [comment]
I am using aiogram and flet libraries. How to open flet app on Telegram Bot webview? |
I tried it:
flet app  ``` app(target=main, assets_dir='assets', port=8550, view=AppView.WEB_BROWSER) ```

open on webview ``` 
keyboard.add(InlineKeyboardButton(text="Open site", web_app=WebAppInfo(
        url="https://000.0.0.0:8550/")))
    await message.answer("Menu inline button", reply_markup=keyboard) ```

How to open webview without creating new port? How to open as an app?

**Comment by ndonkoHenri** [comment]
Please can you build a simple example so we can try?

**Comment by judeetuka** [comment]
I am currently experimenting with Flet and python-telegram-bot. I want to see if I can use Flet as the webhook for python-telegram-bot (I think this is what you mean). I found this regarding python-telegram-bot webhooks - [**HERE**](https://docs.python-telegram-bot.org/en/v21.6/examples.customwebhookbot.html).

I will post an update as it goes.
